src/molecules/formfields/SliderField/index.js

```diff
@@ -28,17 +30,17 @@ class SliderField extends Component {
       sliderValues,
     } = this.props;
 
+    const { sliderIndex } = this.state;
+
     this.setState({ sliderIndex: value }, () => {
-      const sliderAmount = sliderValues[this.state.sliderIndex];
+      const sliderAmount = sliderValues[sliderIndex];
 
       return input.onChange(sliderAmount);
     });
   }
 
   findClosestAmount(values, value) {
-    return values.reduce((prev, curr) =>
-      Math.abs(curr - value) < Math.abs(prev - value) ? curr : prev
-    );
+    return values.reduce((prev, curr) => Math.abs(curr - value) < Math.abs(prev - value) ? curr : prev);
```

Comment:
if you add braces to this function you can keep it as two lines, the eslint `arrow-body-style` rule enforces this

Reply:
changed